blocks/api/registration.js

```diff
@@ -32,6 +32,7 @@ export function registerBlockType( slug, settings ) {
 		return;
 	}
 	if ( ! /^[a-z0-9-]+\/[a-z0-9-]+$/.test( slug ) ) {
+		console.log( slug );
```

Comment:
Gah, was tracking down which block was throwing an error... We should probably add this as a rule to the linter, `npm run test` didn't pick it up